apps/arweave/src/ar_http.erl

```diff
@@ -301,17 +300,21 @@ method_to_list(_) ->
 	"unknown".
 
 request(PID, Args) ->
-	Timer = inet:start_timer(maps:get(timeout, Args, ?HTTP_REQUEST_SEND_TIMEOUT)),
+	Timeout = maps:get(timeout, Args, ?HTTP_REQUEST_SEND_TIMEOUT),
```

Comment:
I realize I'm not sure how the gun request/response works. We'll often pass in a `timeout => Timeout` option when making an HTTP request. I see that that gets pass into the `request2` call ... which is where I assume it was used. But I guess the gun works, waiting on the response is handled separately, so we also need to pass the timeout into the response wait? So that we correctly timeout if no response is handled in time?

Previously we used the inet_timer, but now we're just using the same `timeout => Timeout` option?

If I've understood correctly: sounds good!

Reply:
Got it! LGTM. Merge when ready.